packages/vapor/resources/icons/svg/source-amazonS3.svg

```diff
@@ -1 +1 @@
-<svg enable-background="new 0 0 32 32" viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg"><path d="m30 2v28h-28v-28zm0-2h-28c-1.1 0-2 .9-2 2v28c0 1.1.9 2 2 2h28c1.1 0 2-.9 2-2v-28c0-1.1-.9-2-2-2z" fill="#bcc3ca"/><path d="m25.6 9.3v13.5l-1.6.7v-15z" fill="#e05243"/><g fill="#8c3123"><path d="m16 10.5v11.1l8 1.9v-15z"/><path d="m6.4 9.3v13.5l1.6.7v-15z"/></g><path d="m16 10.5v11.1l-8 1.9v-15z" fill="#e05243"/><g fill="#8c3123"><path d="m12.5 6.2v4.9l3.5-.8v-5.9z"/><path d="m12.5 20.9v4.9l3.5 1.8v-5.9z"/><path d="m12.5 13.5v4.9l3.5.5v-5.8z"/></g><g fill="#e05243"><path d="m19.5 13.5v4.9l-3.5.5v-5.8z"/><path d="m19.5 6.2v4.9l-3.5-.8v-5.9z"/><path d="m19.5 20.9v4.9l-3.5 1.8v-5.9z"/></g><path d="m16 10.3-3.5.8 3.5.7 3.5-.7z" fill="#5e1f18"/><path d="m16 20.2-3.5.7 3.5.8 3.5-.8z" fill="#f2b0a9"/></svg>
\ No newline at end of file
+<svg viewBox="0 0 64 64" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="M0 0h64v64H0z"/><g fill-rule="nonzero"><path fill="#E05243" d="M51.2 18.6v27L48 47V17z"/><g fill="#8C3123"><path d="M32 21v22.2L48 47V17zM12.8 18.6v27L16 47V17z"/></g><path fill="#E05243" d="M32 21v22.2L16 47V17z"/><g fill="#8C3123"><path d="M25 12.4v9.8l7-1.6V8.8zM25 41.8v9.8l7 3.6V43.4zM25 27v9.8l7 1V26.2z"/></g><g fill="#E05243"><path d="M39 27v9.8l-7 1V26.2zM39 12.4v9.8l-7-1.6V8.8zM39 41.8v9.8l-7 3.6V43.4z"/></g><path fill="#5E1F18" d="M32 20.6l-7 1.6 7 1.4 7-1.4z"/><path fill="#F2B0A9" d="M32 40.4l-7 1.4 7 1.6 7-1.6z"/></g></g></svg>
```

Comment:
I fully understand what I'm currently reading! I'll just wait to see the demo just in case (but I have a clear picture of what each icon look like).

Reply:
You also can have a quick look at the result directly if you didn't already know 
![image](https://user-images.githubusercontent.com/35579930/104056761-ff12f280-51be-11eb-8aa2-ae4e2c62c501.png)

Reply:
Oh! Thanks! (Still a lot to learn).